sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/HadoopFileLinesReader.scala

```diff
@@ -51,7 +51,6 @@ class HadoopFileLinesReader(
       new Path(new URI(file.filePath)),
       file.start,
       file.length,
-      // TODO: Implement Locality
```

Comment:
maybe we should leave some comment to explain where the locality is decided.

Reply:
ok, thanks